### ISSUE TITLE
Command have unapproved verbs

### ISSUE THREAD

**Issue by wsmelton** [akamai/akamaipowershell#31]
Importing the module into a PowerShell session will always spit a warning out when there are verbs used on command names that are not part of the approved list from Microsoft/PowerShell team. It is a general practice to stick with the approved verbs that are found via `Get-Verb`.

A common practice to get around it and ensure the warning messages are not observed by users is to create the command in the module using an approved verb, and add the desired command name as an alias.

Output received:
```
[2] > ipmo C:\Modules\AkamaiPowershell\
WARNING: The names of some imported commands from the module 'AkamaiPowershell' include unapproved verbs that might make them less discoverable. To find the commands with unapproved verbs, run the Import-Module command again with the Verbose parameter. For a list of approved verbs, type Get-Verb.
```

**Comment by stuartio** [comment]
So this came up early in the module's history, and the feeling was that the approved verbs didn't fit into all of our scenarios (e.g. activate, purge etc.). So we took the decision to use verbs which more closely match what the function is doing but at the cost of spitting out that irritating warning. We could I guess change those functions to use approved verbs and keep the previous function name as an alias, but I worry it would make the code harder to troubleshoot since currently the filename is the function name so finding the source is quite easy.

Curious as to how other modules have solved this issue, if they have scenarios where the approved verbs don't quite fit.

**Comment by wsmelton** [comment]
Not sure how long back that history goes as I have not seen the current approved verb list not cover a command's primary action. I don't know that it is really an issue to solve directly because majority of module authors just work with the approved verbs.

Purge --> Remove
Activate --> Publish (or Register)
List --> Get

**Comment by stuartio** [comment]
So as I understand the list of approved verbs is there to make functions more discoverable for users. However. the approach we have used is to use the verbs (mostly) that are referenced in the Akamai API docs, and are used within the Akamai Control Centre UI.

So, while Publish-Property would be similar enough to Activate-Property, since "activate" is the verb generally used when discussing that operation with Akamai I would argue that using Publish would make the function _less_ discoverable. There are certainly some we could do, List- for example, though my main reasons for that one were a) List is how it the operation is described in the docs, and b) a Get-<something> command (singular) would only differ from its Get-<somethings> plural equivalent by a single letter, so would be easier to make a mistake.

Some of the functions (most of the ones in the Shared directory) which have unknown verbs we could simply not export (since they aren't really meant to be used directly) but others, such as the NSAPI functions which are 1 to 1 with the actual operation names, it would definitely make it harder to find if we changed the names. 

Have you found the functions to be difficult to discover using the unapproved verbs? Does anyone else who might be reading this feel strongly one way or the other?

We can certainly look at some kind of sanitisation of naming structure for v2 when it comes along.
